config/application.rb

```diff
@@ -23,13 +22,28 @@ module Shipit
   class Application < Rails::Application
     Pubsubstub.use_persistent_connections = false
     # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 6.0
+    config.load_defaults 7.1
+    config.read_encrypted_secrets = false
 
     config.active_job.queue_adapter = :sidekiq
 
-    # Settings in config/environments/* take precedence over those specified here.
-    # Application configuration can go into files in config/initializers
-    # -- all .rb files in that directory are automatically loaded after loading
-    # the framework and any gems in your application.
+    # Please, add to the `ignore` list any other `lib` subdirectories that do
+    # not contain `.rb` files, or that should not be reloaded or eager loaded.
+    # Common ones are `templates`, `generators`, or `middleware`, for example.
+    config.autoload_lib(ignore: %w(assets tasks))
+
+    # Configuration for the application, engines, and railties goes here.
+    #
+    # These settings can be overridden in specific environments using the files
+    # in config/environments, which are processed later.
+    #
+    # config.time_zone = "Central Time (US & Canada)"
+    # config.eager_load_paths << Rails.root.join("extras")
+  end
+
+  # This is to override the `secrets` global that for some reason doesn't work
+  def secrets
+    Rails.application.secrets
   end
```

Comment:
This needed to be added. Somehow the `secrets` wouldn't be available otherwise, this might be part of rails' transition to encoded secrets